### DISCUSSION TITLE
Need Plumber 2 Licene

### DISCUSSION THREAD

**Discussion by nnahmed** [nathanwoulfe/Plumber-2#146] [General]
Dear Nath

I need the Plumber 2 license copy, for Umbraco version 8.13. Please help me out.

Thanx

**Comment by basselissa** [comment]
Dear Nathan,

Umbraco are not selling the license for earlier than v10,  do you still support Umbraco v8 with Plummer.workflow v2?

**Comment by nathanwoulfe** [comment]
Hi both. Short answer - no, licenses are not available for Workflow or Plumber on Umbraco 8. 

Plumber will only receive security fixes, and will be EOL in February 2025 (same time as Umbraco 8). All versions of Plumber should be considered legacy.
